build.sbt

```diff
@@ -95,8 +95,8 @@ runStandalone := {
   (Compile / run).evaluated
 }
 
-(Compile / compile) := ((Compile / compile) dependsOn generateCode).value
-(Test / test)       := ((Test / test) dependsOn generateCode).value
+(Compile / compile) := ((Compile / compile) dependsOn generateCode).value //
+(Test / test)       := ((Test / test) dependsOn generateCode).value //
```

Comment:
?

Reply:
Ops, coding sometimes I set generateCode as comment, and I forgot to clean